scripts/gke/deploy.sh

```diff
@@ -159,6 +159,10 @@ if ${KUBEFLOW_DEPLOY}; then
 
   # Install the GPU driver. It has no effect on non-GPU nodes.
   kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/stable/nvidia-driver-installer/cos/daemonset-preloaded.yaml
+
+  # Install Stackdriver Kubernetes agents.
```

Comment:
Why do we need to do this? Aren't the stackdriver agents installed by default?

Reply:
The beta instructions don't mention installing these agents
https://cloud.google.com/monitoring/kubernetes-engine/installing?

Reply:
The instruction there are using gcloud or pantheon. We are doing manually here.

Reply:
Should we take advantage of the YAML support in ksonnet and start a package for all the YAML specs for GKE?

We could do this in a follow on a PR. I think the question is how can we make it easier to keep in sync between our web app and deploy.sh

Right now we'd have to specify the YAML spec in two different locations.

Reply:
https://github.com/kubeflow/kubeflow/issues/1288